magefile.go

```diff
@@ -32,7 +32,7 @@ var (
 
 	publicDir    = "./public"
 	buildDir     = "./build"
-	packagePaths = []string{"./dev/packages/generated/", "./dev/packages/example/"}
+	packagePaths = []string{"./dev/packages/generated/", "./dev/packages/example/", "./dev/packages/beats/"}
```

Comment:
I'm thinking if we should add it here already or not. I like that it does validate the packages. But I'm wondering if they should already served. The good news is I assume things like nginx and system package keep working, as they have a never version number.

Reply:
@mtojek To get this PR in, can you remove this and merge it without it?

Reply:
Sure, not a problem. Reverted.